rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json

```diff
@@ -80,6 +80,10 @@
     "idref": {
       "title": "IdRef authorities",
       "type": "object"
+    },
+    "deleted": {
+      "title": "Deletion date",
```

Comment:
```suggestion
    "deletion_date": {
      "title": "Deletion date",
```

Reply:
If I change this I have to change it too in rero-mef. We can open an issue and change it in a new release of rero-mef.

Reply:
There are also date fields named `_updated` and `_changed`.